lib/animina_web/live/flags_live.ex

```diff
@@ -53,10 +53,10 @@ defmodule AniminaWeb.FlagsLive do
     )
     |> assign(
       :opposite_color_flags_selected,
-      fetch_flags(socket.assigns.current_user.id, :white)
-      |> Enum.map(fn flag -> flag.flag.id end)
+      fetch_flags(socket.assigns.current_user, :white)
```

Comment:
I don't like the name `fetch_flags/2`. For me this feels more like a `filter_flags/2`.